industry_benchmarks/utils/plan_rbfe_network.py

```diff
@@ -0,0 +1,227 @@
+import string
+import click
+import pathlib
+import logging
+import os
+from functools import partial
+from openff.units import unit
+import openfe
+from openfe.protocols.openmm_rfe.equil_rfe_methods import RelativeHybridTopologyProtocol
+from rdkit import Chem
+import kartograf
+from openff.toolkit import (
+    RDKitToolkitWrapper, AmberToolsToolkitWrapper
+)
+from openff.toolkit.utils.toolkit_registry import (
+    toolkit_registry_manager, ToolkitRegistry
+)
+
+logger = logging.getLogger(__name__)
+
+amber_rdkit = ToolkitRegistry([RDKitToolkitWrapper(), AmberToolsToolkitWrapper()])
+
+
+def gen_charges(smc):
```

Comment:
@IAlibay to work out if we can set the residue name here, maybe set it to "LIG"

Reply:
This isn't possible right now because of how we serialize our SMCs: https://github.com/OpenFreeEnergy/gufe/issues/327